graphql/database/experiments/features.js

```diff
@@ -43,30 +43,60 @@ const features = {
 
 features[YAHOO_SEARCH_EXISTING_USERS] = {
   defaultValue: false,
-  // Enable when SFAC search engine is enabled.
-  // rules: [
-  //   {
-  //     condition: {
-  //       isTabTeamMember: true,
-  //       env: 'local',
-  //     },
-  //     force: true,
-  //   },
-  // ],
+  rules: [
+    /* Begin internal overrides */
+    {
+      condition: {
+        v4BetaEnabled: true,
```

Comment:
I think (but am not 100% confident) this has to use `$eq` instead because a simple `true` is truthy:
https://github.com/gladly-team/tab/pull/1235/files

Reply:
If you click into that link, `v4BetaEnabled` in that condition is also `true`

Reply:
Is the truthy check just the case for that string key?

Reply:
@jedtan Sorry, I was unclear. That PR link was just to show the use of `$eq: true` in the override attribute, which I'm suggesting we use anywhere we have a boolean match (even though the linked PR doesn't do so for v4BetaEnabled).

The behavior I observed was that setting an attribute value of `false` also matched on a simple `true` statement—so it's actually not "truthy" matching but whether or not the value is defined (?). I couldn't find clarity in the MongoDB docs, so this was just trial and error, and I'm not totally certain I'm right.

If I'm right, v4BetaEnabled matching wouldn't work at all (it would match any defined value). This doesn't matter right now because legacy doesn't request experiment info from the endpoint but would break in the future.